docs/manuals/provisioning-solr/provisioning-solr.md

```diff
@@ -0,0 +1,171 @@
+# Provisioning SOLR
+
+***
+> **Colofon** <br>
+> Datum : 18-6-2025 <br>
+> Versie :   1.0.0 <br>
+> Verandering : provisioning solr <br>
+> Toegangsrechten : Alleen lezen <br>
+> Status : Definitief <br>
+> Redacteur : John Bol <br>
+> Auteur(s) : John Bol <br>
+****
+
+Versiegeschiedenis:
+
+| 0.1   | Initiële versie                                                                                                                            |
+|-------|--------------------------------------------------------------------------------------------------------------------------------------------|
+
+<div style="page-break-after: always"></div>
+
+# Inhoud
+
+[*introduction*](#introduction)
+[*requirements*](#requirements)
+[*configuration*](#configuration)
+[*provisioning*](#provisioning)
+
+<div style="page-break-after: always"></div>
+
+#  Introduction
+
+originally solr was included in the helm chart of zaakafhandelcomponent. However, the current version of solr was not properly supported. Therefore the solr configuration part was deprecated in favor of provisioning solr seperately. This page describes how to configure solr for use by zaakafhandelcomponent.
+Update: due to stability issues we have moved to solrcloud in k8s
+Update: the zac helm charts can provision solrcloud for you but not the solr-operator crds. When used you will still have to create the zac solr core yourself.
+
+#  Requirements
+
+this requires a working helm installation in order to generate the required templates
+the generated templates are too large to use kubectl apply
+
+# Configuration
+
+the zac core by default uses the standard solrconfig.xml provided by solr
+this configuration file is managed by zookeeper and must be changed there if this is required. More information about that can be found here:
+https://solr.apache.org/guide/6_6/using-zookeeper-to-manage-configuration-files.html
+
+you can view the standard settings here:
+https://solrcloud-common-zac-dev.dimpact.lifely.nl/solr/#/zac/files?file=solrconfig.xml
```

Comment:
This is not accessible or dead.

Reply:
Maybe make it a link with a <base url> in it? so that it's not a complete link anyway...